rero/rero-ils-ui
# general: update to font-awsome-6.

Co-authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## How to test?

We need to check all pages where some icons could be used to check if icons are correctly displayed.
![image](https://user-images.githubusercontent.com/10031585/163974054-0045411d-8eb2-45d8-98b0-0b3ddeeb987a.png)


## Code review check list
